internal/cli/events/projects_list_test.go

```diff
@@ -87,5 +86,5 @@ func Test_projectListOpts_Run_WithInvalidDate(t *testing.T) {
 	if err == nil {
 		t.Fatal("Run() expected error")
 	}
-	assert.True(t, strings.Contains(err.Error(), "parsing time"))
+	assert.Contains(t, err.Error(), "parsing time")
```

Comment:
https://pkg.go.dev/github.com/stretchr/testify/assert#ErrorContains

Reply:
Good to know that function exists. Updated!